tests/app/celery/test_provider_tasks.py

```diff
@@ -5,41 +5,74 @@
 
 import app
 from app.celery import provider_tasks
-from app.celery.provider_tasks import deliver_sms, deliver_email
+from app.celery.provider_tasks import deliver_sms, deliver_throttled_sms, deliver_email
 from app.clients.email.aws_ses import AwsSesClientException
 from app.exceptions import NotificationTechnicalFailureException
 
 
+sms_methods = [
+    (deliver_sms, 'deliver_sms'),
+    (deliver_throttled_sms, 'deliver_throttled_sms'),
+]
```

Comment:
Used `@pytest.mark.parametrize` on every test method which was using `deliver_sms` before to prove that the 2 methods have the same logic